CompanyApiTest/CompanyApiTest.cs

```diff
@@ -68,6 +69,190 @@ public async Task Should_return_bad_reqeust_when_create_company_given_a_company_
             Assert.Equal(HttpStatusCode.BadRequest, httpResponseMessage.StatusCode);
         }
 
+        [Fact]
+        public async Task Should_return_all_companies_list_when_get_all_companies()
+        {
+            // Given
+            await ClearDataAsync();
+            Company companyGiven = new Company("slb");

```

Comment:
To Improve: Better use `CreateCompanyRequest` and `CreateEmployeeRequest` since FE doesn't provide ID